src/service_objects/game_piece_action_resolvers/move_resolver.ts

```diff
@@ -63,5 +146,32 @@ export default async function resolveMoveAction(
   action.actionData = newActionData;
   await action.save({ transaction });
 
+  // Warn on console if the state does not match
+  if (snarkySuccess) {
```

Comment:
If applying the move worked, then check that recreating the state from the game id agrees - haven't gotten this to work yet.